opt4j-core/src/main/java/org/opt4j/core/common/random/MTRandom.java

```diff
@@ -32,8 +32,8 @@
  *         <p>
  *         A Java implementation of the MT19937 (Mersenne Twister) pseudo random
  *         number generator algorithm based upon the original C code by Makoto
- *         Matsumoto and Takuji Nishimura (see <a
- *         href="http://www.math.sci.hiroshima-u.ac.jp/~m-mat/MT/emt.html">
+ *         Matsumoto and Takuji Nishimura (see

```

Comment:
Shouldn't this be handled with the new Citation annotation?

Reply:
I suggest separation of concerns and fix this in another patch to restrict this to codacy styling.

Reply:
I did intentionally not use @citation here as the above references the code source, not a paper. Nonetheless, we can discuss of the corresponding paper should also be cited here, but this is SoC.